cmd/dagger/functions.go

```diff
@@ -441,10 +441,11 @@ func (fc *FuncCommand) addSubCommands(cmd *cobra.Command, dag *dagger.Client, fn
 
 func (fc *FuncCommand) makeSubCmd(dag *dagger.Client, fn *modFunction) *cobra.Command {
 	newCmd := &cobra.Command{
-		Use:     cliName(fn.Name),
-		Short:   strings.SplitN(fn.Description, "\n", 2)[0],
-		Long:    fn.Description,
-		GroupID: funcGroup.ID,
+		Use:                   cliName(fn.Name),
```

Comment:
Don't you need to add `[options]` here?

Reply:
You might be right, but given that we currently cannot interpret the `--help` after a module's function in the `functions` command, I did not see the need to implement it right away (I might be wrong and might be testing improperly).

```shell
./dagger -m github.com/levlaz/daggerverse/mariadb@v0.2.1 functions base --help 
✔ initialize 1.6s

Error: no function '--help' in object type 'Container'
```

and if putting the help before, I have the same usage as with `dagger functions --help`:
```shell
 ./dagger -m github.com/levlaz/daggerverse/mariadb@v0.2.1 functions --help base
List available functions in a module.

This is similar to `dagger call --help`, but only focused on showing the
available functions.

USAGE
  dagger functions [options] [function]...

OPTIONS
  -m, --mod string   Path to dagger.json config file for the module or a directory
                     containing that file. Either local path (e.g. "/path/to/some/dir") or a
                     github repo (e.g. "github.com/dagger/dagger/path/to/some/subdir")

INHERITED OPTIONS
  -d, --debug             show debug logs and full verbosity
      --progress string   progress output format (auto, plain, tty) (default "auto")
  -s, --silent            disable terminal UI and progress output
```

Reply:
But this code doesn't apply to `functions` right? It's for `call` subcommands. In `functions` it's positional arguments.

Reply:
> But this code doesn't apply to `functions` right? It's for `call` subcommands. In `functions` it's positional arguments.

You are right, my bad. It is indeed required for `dagger call` with two functions:
```shell
./dagger --progress=plain -m github.com/levlaz/daggerverse/mariadb@v0.2.1 call base workdir --help
Retrieves the working directory for all commands.

USAGE
  dagger call base workdir [options]
```